coderdojochi/urls.py

```diff
@@ -374,3 +341,44 @@
     if 'debug_toolbar' in settings.INSTALLED_APPS:
         import debug_toolbar
         urlpatterns = [path('__debug__/', include(debug_toolbar.urls))] + urlpatterns
+
+else:
+    from django.contrib.staticfiles.storage import staticfiles_storage
+    from django.contrib.staticfiles.views import serve
+
+    # favicon.ico
+    favicons = [
```

Comment:
Why disable the static pages/assets on debug? Just wondering! :)

Reply:
The django docs suggest not having it on in development so that you don't have to run `collectstatic` every time you want to make a css/js/image change. They also stressed how inefficient it is and to never do that in prod.